internal/storage/storage.go

```diff
@@ -138,13 +139,18 @@ func (s *ConfigMaps) buildObject(obj client.Object, owner client.Object) (*corev
 	}
 	gvk := obj.GetObjectKind().GroupVersionKind()
 
+	objName := obj.GetName()
+	if strings.Contains(objName, ":") {
```

Comment:
Would it make more sense to just let this fail? I ask because this could probably grow to include the remaining things that it takes to validate a label. Alternatively, this could go into an internal package and house more expanded logic. We may not care to make this too elaborate but just want to check that we want this logic to begin with.

Reply:
I think erroring out here is the wrong thing to do as the Bundle contents are technically valid, and the problem is with how we're building up a ConfigMap internally. I guess we could produce a warning-level log, but the value of that seems low, and performing a simple translation so it doesn't violate a metadata.Label validation regex shouldn't increase the maintenance burden all that much.

Reply:
Maybe we just move all of the `object-*` fields to annotations? I don't think these are actually used in selectors anywhere.

Reply:
I think this is reasonable for now, and solves the immediate issue I was having deploying Bundles locally. We can revisit moving some of them back to labels if we need to label select ConfigMaps that reference a specific namespace, or GVK, etc.

Reply:
Done in the latest force-push.